corehq/apps/hqwebapp/templatetags/hq_shared_tags.py

```diff
@@ -595,10 +592,19 @@ def _create_page_data(parser, token, node_slug):
     value = parser.compile_filter(split_contents[2])
 
     class FakeNode(template.Node):
+        # must mock token or error handling code will fail and not reveal real error
+        token = Token(TokenType.TEXT, '', (0, 0), 0)
```

Comment:
This is what addresses the "NoneType has no attribute position" error.

Reply:
Thanks, this clears up my confusion. I saw the comment, but didn't understand that this resolves the `AttributeError`.